services/tenant-ui/frontend/src/store/reservationStore.ts

```diff
@@ -0,0 +1,59 @@
+import { API_PATH } from '@/helpers/constants';
+import axios from 'axios';
+import { defineStore, storeToRefs } from 'pinia';
+import { ref } from 'vue';
+import { useConfigStore } from './configStore';
+
+export const useReservationStore = defineStore('reservation', () => {
+  const { config } = storeToRefs(useConfigStore());
+  // A raw api call without using the interceptors from the acapyApiStore
+  // Needed for the open call to reservation at this point
+  const api = axios.create({
+    baseURL: config.value.frontend.tenantProxyPath,
+  });
+
+  // state
+  const loading: any = ref(false);
+  const error: any = ref(null);
+  const reservation: any = ref(null);
+
+  // actions
+  async function makeReservation(payload: any = {}) {
+    console.log('> reservationStore.makeReservation');
+    error.value = null;
+    loading.value = true;
+    console.log(payload);
+    await api
```

Comment:
Can't use the API store that all the other calls use since that results in a blank Bearer header from the interceptor and that 401s you.

Reply:
ahhh, nice.